policyDefinitions/Monitoring/deny-change-of-retention-days-in-log-analytics-workspace/azurepolicy.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "c7eedeb9-bf27-40fa-91ce-4ccb1ae35edc",
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "properties": {
+    "displayName": "Deny change of retention days in Log Analytics workspace",
```

Comment:
Please change the display name as it's confusing that it says "Deny...", but the policy can be configured to other effects too.